src/admin/__tests__/fetchTopicOffsets.spec.js

```diff
@@ -37,22 +38,5 @@ describe('Admin', () => {
         'Invalid topic null'
       )
     })
-
-    test('returns the current topic offset', async () => {
```

Comment:
Why was this test removed? Most of the functionality is tested within the cluster module, but we still need to test how it's exposed through the admin interface.

Reply:
Ah, because it was moved to `fetchTopicOffsetsByTimestamp.spec.js`. I don't agree with that move, since that spec file is explicitly testing fetching offsets by timestamp.

Reply:
Oh, sorry, will restore that file.